doc/tech/approach.md

```diff
@@ -119,7 +119,7 @@ VTEP could be manually provisioned in the Underlay Networking or be automated an
 
 When there are multiple simultaneous SW Virtualization Layers on the same HW Infrastructure, there is a need to ensure Underlay networking separation in the HW Infrastructure Layer. This separation can be done manually through provisioning of a statically configured separation of the Underlay networking in the HW Infrastructure Layer. A better and more agile usage of the HW Infrastructure is to have an authoritative SDN provisioning controller function (here denoted SDNuP) that can be controlled through an automation interface from a HW Infrastructure Orchestrator. The main tasks for the SDNuP are to discover and establish the Underlay resources and then ensure separation of the shared HW Infrastructure Underlay networking resources.
 
-Multiple instances of a Containerized Virtualization Layer (CaaS) running on an Infrastructure as a Service (IaaS) Virtualization Layer could make use of the IaaS layer to handle the required Underlay Networking separation. In these cases, also the IaaS Virtualization Infrastructure Manager (VIM) could include a SDNu control interface enabling automation.
+Multiple instances of Container Virtualization Layers (CaaS) running on an Infrastructure as a Service (IaaS) Virtualization Layer could make use of the IaaS layer to handle the required Underlay Networking separation. In these cases, also the IaaS Virtualization Infrastructure Manager (VIM) could include a SDNu control interface enabling automation.
```

Comment:
@TFredberg The [Glossary](https://github.com/cntt-n/CNTT/blob/master/doc/tech/glossary.md) defines, "CaaS: Container-as-a-Service". Do you want to reword, "... Container-as-a-Service (CaaS) Virtualisation Layers ...."

In the sentence, "In these cases, also the IaaS ...", the work "also" is unnecessary.

Reply:
Fixed